lib/scripts/controllers/options.js

```diff
@@ -56,8 +59,13 @@ export class OptionCtrl {
       console.log(data);
       this.patterns = JSON.parse(data.savedPatterns);
       this.key = data.yandexTranslatorApiKey;
+
       this.blacklistedWords = data.userBlacklistedWords.slice(1, -1).split('|');
+      this.blacklistedWords = this.blacklistedWords.filter(function(n){ return n != '' });
+
       this.blacklistedWebsites = data.blacklist.slice(1, -1).split('|');
+      this.blacklistedWebsites = this.blacklistedWebsites.filter(function(n){ return n != '' });
```

Comment:
Then this corner case should preferably be handled in line 66, instead of fixed in line 67...  Can't the blacklist be sliced and split in a more careful way, so that this problematic corner case doesn't occur?